horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -1,4 +1,4 @@
-classdef PixelData < matlab.mixin.Copyable
+classdef PixelData < handle
```

Comment:
are there any other consequences swapping from the `Copyable` mixin to being a simple handle class?

Reply:
Not that I can think of, we'll see in testing. The `Copyable` parent class implements a `copy` method that does a shallow copy of all attributes, that's all it does. I've implemented our own `copy` method which instantiates a _new_ object and then shallow copies across the necessary properties.

A consequence of this is that when we add new properties, we'll need to think about whether they need to be copied in the copy-constructor, or whether the normal constructor can handle it.

I decided this way was better since you cannot override the `copy` method implemented by `Copyable`. So I couldn't easily add the `arg.tmp_io_handler_.copy_folder(obj.object_id_);` step. There may have been a way using [`copyElement`](https://uk.mathworks.com/help/matlab/ref/matlab.mixin.copyable-class.html), but it felt simpler to do what I did here.

Reply:
It may be worth adding that note as a comment either in the `copy()` method or in the code block where new properties would be added.